bookmarks.html

```diff
@@ -14,99 +14,158 @@
     <title>Quizzzle</title>
   </head>
   <body>
-    <header class="zz-top">
+    <header class="zz-top title">
       <h1>Bookmarks</h1>
     </header>
 
     <main>
       <ul class="card-container" role="list">
-        <li class="card">
-          <div class="question-container">
-            <h2>Question</h2>
-            <p class="question">
+        <li class="card q-card">
+          <div class="q-card__q-container">
```

Comment:
Why not `card__container`? Or `quiz-card__container`? Or `card__question`? I don't know what `q` stands for: question or quiz?